File: JS/Sequencial/EX12_A.js
/**
 * 12.faca um programa que receba dois numeros maiores que zero, calcule e mostre um elevado ao outro.
 */

alert("Calculando um numero elevado a outro ")
var numero1 = prompt("Digite o primeiro numero")
var numero2 = prompt("Digite o segundo numero")

numero1 = parseFloat(numero1)
numero2 = parseFloat(numero2)

var resposta1 = Math.pow(numero1, numero2)
var resposta2 = Math.pow(numero2, numero1)

console.log("Reposta 1: "+ resposta1 +" | "+ "Reposta 2: "+ resposta2)

File: python/Sequencial/EX12_A.py
#12.faca um programa que receba dois numeros maiores que zero, calcule e mostre um elevado ao outro.

import math;

print("***********************SEQUÊNCIAL*****************************")
print("**********************Programa*12A****************************")
print("*************Dois numeros elevados um ao outro****************")

num1 = float(input("Digite o primeiro numero: "))
num2 = float(input("Digite o segundo numero: "))

rs1 = math.pow(num1, num2)
rs2 = math.pow(num2, num1)

print('Resposta 1: {0}  | Resposta 2: {1}'.format(rs1, rs2))

